x-pack/plugins/ml/public/alerting/jobs_health_rule/tests_selection_control.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiFormFieldset, EuiSpacer, EuiSwitch } from '@elastic/eui';
+import { JobsHealthRuleTestsConfig } from '../../../common/types/alerts';
+import { getResultJobsHealthRuleConfig } from '../../../common/util/alerts';
+
+interface TestsSelectionControlProps {
+  config: JobsHealthRuleTestsConfig;
+  onChange: (update: JobsHealthRuleTestsConfig) => void;
+}
+
+export const TestsSelectionControl: FC<TestsSelectionControlProps> = ({ config, onChange }) => {
+  const uiConfig = getResultJobsHealthRuleConfig(config);
+
+  const updateCallback = useCallback(
+    (update: Partial<Exclude<JobsHealthRuleTestsConfig, undefined>>) => {
+      onChange({
+        ...(config ?? {}),
+        ...update,
+      });
+    },
+    [onChange, config]
+  );
+
+  return (
+    <EuiFormFieldset
+      legend={{
+        children: i18n.translate(
+          'xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.legend',
+          {
+            defaultMessage: 'Select health checks to perform',
+          }
+        ),
+      }}
+    >
+      <EuiSwitch
```

Comment:
Just to confirm, is it a reasonable use case that the user might create or edit a rule so that all the checks are disabled, or should we be erroring if there are no active selections?

![image](https://user-images.githubusercontent.com/7405507/126322137-25cf4263-962a-45bc-a000-35dc6f34ebe6.png)

Reply:
I actually just added a validation that enforces the selection of at least one health check. Important moment - it will not validate alerting rules you created before this commit.

Reply:
![image](https://user-images.githubusercontent.com/5236598/126323756-74cfa420-fcd8-4941-945d-167ce47a9779.png)